### ISSUE TITLE
Nother disappearing card

### ISSUE THREAD

**Issue by alexiskennedy** [weatherfactory/cultistsimulator-visible#77]
[Vaevictusnet]
"used C to collect the job, post work, and then
[Cultist Simulator.zip](https://github.com/weatherfactory/cultistsimulator-visible/files/8235880/Cultist.Simulator.zip)
drug it from the desktop to the work ... and missed to the left... still on the action popup but off to the left. I didn't have recording on or i'd supply it. save file attached."

**Comment by Vaevictusnet** [comment]
[2022-03-11 18-13-31-1.zip](https://github.com/weatherfactory/cultistsimulator-visible/files/8236670/2022-03-11.18-13-31-1.zip)
got a video this time.

**Comment by alexiskennedy** [comment]
Thanks! I can't repro, even with this :) - it must be some pixel placement edge case thing which the Perlin card wiggle makes unpredictable. but there is a suggestive error in your player.log, and it gives me a good direction to look in.

Key q: have you seen it happen where there isn't a child slot (like Diligence) in the mix? I'm guessing yes?

**Comment by Vaevictusnet** [comment]
Yes, I believe I have. honestly i've been waggling with glover placement each time and it definitely isn't an everytime thing.  Is there a condition where glover job has the diligence on a subsequent page still?  I really feel like the first re-report of this bug was on a page without diligence.

**Comment by alexiskennedy** [comment]
I *think* this was fixed with 'Esau' (#82) but I'm not certain enough to close it. Depri for now.

**Comment by alexiskennedy** [comment]
Another depri. No further reports bar a couple of angry people on Steam who didn't provide deets.
